qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,30 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def _session_client_evict(self, cmd):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.rank_asok(cmd + ['ls'])
+        self.assertIsNotNone(info)
+        size = len(info)
+        try:
+            info = self.fs.rank_asok(cmd + ['evict'])
+        except CommandFailedError as e:
+            self.assertEqual(e.exitstatus, -errno.EPERM)
+        else:
+            raise AssertionError("Expected EPERM")
```

Comment:
IMO this would be better expressed with:
```
with self.assertRaises(CommandFailedError) as ctx:
    self.fs.rank_asok(cmd + ['evict'])
self.assertEqual(ctx.exception.exitstatus, -errno.EPERM)
```

Reply:
Sure, looks better and neat.